storage/database.go

```diff
@@ -250,6 +250,13 @@ func (d *db) Close() error {
 	d.state = databaseClosed
 
 	// Close database-level resources.
+	if err := d.mediator.Close(); err != nil {
+		return err
+	}
+
+	if err := d.Options().PersistManager().Close(); err != nil {
```

Comment:
This may need to wait till the current ticking finishes (which may perform a flush). Is that the case at the moment?

Reply:
On a second thought this probably should just be removed, as closing the manager will cause the it to panic if it's in the middle of a flush, and all closing does is just setting some nil pointers (which doesn't matter much if the database is shutting down).

Reply:
Removing the close completely.